apps/meteor/client/components/voip/room/VoipRoomForeword.tsx

```diff
@@ -1,15 +1,18 @@
-import type { IRoom } from '@rocket.chat/core-typings';
+import type { IVoipRoom } from '@rocket.chat/core-typings';
 import { Avatar, Box, Tag } from '@rocket.chat/fuselage';
 import { useTranslation } from '@rocket.chat/ui-contexts';
 import React, { ReactElement } from 'react';
 
 import { getUserAvatarURL } from '../../../../app/utils/client';
+import { parseOutboundPhoneNumber } from '../../../../ee/client/lib/voip/parseOutboundPhoneNumber';
 
-export const VoipRoomForeword = ({ room }: { room: IRoom }): ReactElement => {
+export const VoipRoomForeword = ({ room }: { room: IVoipRoom }): ReactElement => {
 	const t = useTranslation();
 
 	const avatarUrl = getUserAvatarURL(room.name || room.fname);
 
+	const roomName = room.name || room.fname;
+
 	return (
```

Comment:
you changed the IVoipRoom and added `name: string;` as required, but looking at the code, we dont look very confident about it(always using name || fname), could you check this for me please?